OptionD-ExpandedSimple/README.md

```diff
@@ -0,0 +1,5 @@
+# Option 4
+
+Simplest option that should let you copy/paste your function.json config into a typescript/javascript file. For that reason, it does not have it's own "new" types folder - it is still using the old types package.
+
```

Comment:
Can you add a `types/index.d.ts` file to this option? I feel like that file is important for defining the expected contract of the prototype. Feel free to look at the other options for examples